### ISSUE TITLE
Install instructions didn't work for me: `honeybee_energy` not `honeybee-energy`

### ISSUE THREAD

**Issue by corymosiman12** [ladybug-tools/honeybee-energy#946]
This might not be relevant for everyone, and I'm not sure what the recommended platforms are for installing, but this is what worked for me:
- macOS 12.6.2
- Macbook Pro M1, 2021

I did:
- `pip3 install -U honeybee-energy`
- Restart/re-source shell
- `python3 -m honeybee_energy` instead of `honeybee-energy`

Not sure why the underscore instead of dash, wasn't totally obvious to me though so I'll add this in case it's useful for anybody else.

**Comment by mostaphaRoudsari** [comment]
Thank you for bringing this to our attention @corymosiman12.

I don't have access to mac but I tried it on Ubuntu (WSL) and `honeybee-energy` worked.

![image](https://user-images.githubusercontent.com/2915573/221298689-ca2ab99c-7468-469b-9dd4-7438a58583bc.png)

I imagine they should be calling the same endpoint from PyPI to get the packages. @chriswmackey we might want to add a note to the README file, just in case.

**Comment by corymosiman12** [comment]
The PyPI endpoint wasn't the issue. The issue was how to actually run it once installed. Your docs say:

> To check if the command line interface is installed correctly use `honeybee-energy --help`

Whereas, to run it from the command line for me, I had to do: `python3 -m honeybee_energy`. Maybe I did something wrong.

**Comment by chriswmackey** [comment]
I'm sorry for the incredibly late response here. I believe that the issue is the result of the `-m` option. As the Python `--help` text states:

```
-m mod : run library module as a script
``

This ensures that the python interpreter uses the module (or in this case package) name instead of the actual CLI name or PyPI name. And, the library is `honyebee_energy` as you can see in the site-packages folder:
![image](https://github.com/ladybug-tools/honeybee-energy/assets/5567574/bfd8b589-f401-483a-9bb0-0249ea553297)

... and it kinda needs to be structured like this in order for the package to be imported with the `import honeybee_energy` statement (since Python will interpret a `-` as a minus sign).

Long story short, if you just call the executable directly by navigating your shell to the Python Scripts folder at `python\Scripts`, then you can call the CLI with `honeybee-energy` instead of `honeybee_energy`.

I hope that answers the question. I'm going to close this issue but feel free to re-open it if you think there's a better solution.
